forecasttools/daily_to_epiweekly.py

```diff
@@ -1,12 +1,24 @@
+"""
+Converts daily resolution dataframes
+to epiweekly dataframes.
+"""
+
 from datetime import datetime
 
+import epiweeks
 import polars as pl
 
 
-# NOTE: this needs to change to ACTUAL epiweeks
-def calculate_epidate(date):
-    iso_calendar = date.isocalendar()
-    return iso_calendar[1], iso_calendar[0]
+def calculate_epidate(date: str):
+    """
+    Converts an ISO8601 formatted
+    date into an epiweek and epiyear.
+
+    date
+        An ISO8601 date.
+    """
+    week_obj = epiweeks.Week.fromdate(datetime.strptime(date, "%Y-%m-%d"))
+    return {"epiweek": week_obj.week, "epiyear": week_obj.year}
 
 
 def daily_to_epiweekly(
```

Comment:
Name could be improved.

Reply:
`week_obj` &rarr; `epiweek` sound good?

Reply:
Something that suggests the function operates on a dataframe.

Reply:
Changed to 

```python
epiweek = epiweeks.Week.fromdate(datetime.strptime(date, "%Y-%m-%d"))
epiweek_df_struct = {"epiweek": epiweek.week, "epiyear": epiweek.year}
return epiweek_df_struct
```

Reply:
My comment was in reference to `def daily_to_epiweekly(`

Reply:
Will change on `main`.